CHANGES.md

```diff
@@ -1057,6 +1057,12 @@ OpenSSL 3.0
 
    *Boris Pismenny*
 
+ * The SSL option SSL_OP_CLEANSE_PLAINTEXT is introduced. If that
+   option is set, openssl cleanses plaintext bytes after delivering
```

Comment:
Probably we should clarify that this means we cleanse *internal* buffers. The application is still responsible for cleansing its own buffers.